app/components/Views/OnboardingCarousel/index.tsx

```diff
@@ -174,6 +177,21 @@ export const OnboardingCarousel: React.FC<OnboardingCarouselProps> = ({
     );
   };
 
+  const termsOfUse = async () => {
```

Comment:
Something doesn't add up, this PR have the same changes of this [one](https://github.com/Web3Auth/metamask-mobile/pull/34/files#diff-3b6b41d6ea8a6f9c2e5f83ef0e285e922904e6ccfbb9dac05e551efeae40b0f5R180-R194) although the branch of this PR is branching off it!

Can we solve this issue to facilitate review? This will make it a bit harder to understand the final result of the code